AboghanemAO/lab_3/lab_3.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+
+
+
+void Sort(char** words, int words_counter) {
+	char* test = words[0];
+	for (int i = 0; i < words_counter; i++) {
+		for (int j = i + 1; j < words_counter; j++) {
+			test = words[i];
+			if ((int)strlen(test) > (int)strlen(words[j])) {
+				words[i] = words[j];
+				words[j] = test;
+			}
+		}
+	}
+}
+
+int main() {
+	char sen[100];
+
+
+	std::cout << "Enter your sentence : ";
+	std::cin.getline(sen, 100);
+
+
+	int words_counter = 1;
+	for (int i = 0; i < (int)strlen(sen); i++) {
+		if (sen[i] == ' ') {
+			words_counter++;
+		}
+	}
+
+	int index = 0;
+	//Create char array of char arrays to store splitted string
+	char** words = new char* [words_counter];
```

Comment:
When you create data with `new `keyword - you also should release memory using `delete `operator because it's leads to memory leak otherwise